salt/modules/kubernetes.py

```diff
@@ -236,18 +236,18 @@ def nodes(**kwargs):
         _cleanup()
 
 
-def node(name, **kwargs):
+def node(name, label_selector='', **kwargs):
     '''
     Return the details of the node identified by the specified name
 
     CLI Examples::
 
-        salt '*' kubernetes.node name='minikube'
+        salt '*' kubernetes.node name='minikube' label_selector='io.kompose.service=nginx'
     '''
     _setup_conn(**kwargs)
     try:
         api_instance = kubernetes.client.CoreV1Api()
-        api_response = api_instance.list_node()
+        api_response = api_instance.list_node(label_selector=label_selector)
```

Comment:
In Salt default params are `None`, not an empty String. That said, it should be `None`. Now, from the description of the kubernetes CoreV1Api class:
```
label_selector: A selector to restrict the list of returned 
objects by their labels. Defaults to everything.
```
Therefore I would write it that way:

```python
def node(name, **kwargs):
    ....
    api_response = api_instance.list_node(**{'label_selector': kwargs.get('label_selector', '')})
```

So then it goes clean and no signature is changed. I don't know whether `label_selector` _has to be_ an empty string or `None` would also work there. In case `None` is fine, then `kwargs.get('label_selector)` is sufficient.

Reply:
The kubernetes client methods fail when label_selector is set to `None` or `False` the only unset value that will work is `""`.  I didn't want to add label_selector to `**kwargs` because that's passed into the client connection setup, but after looking it doesn't look like it will be a problem.